### ISSUE TITLE
Large number of columns in LAU XLSX

### ISSUE THREAD

**Issue by adibaba** [adibaba/LauNuts#1]
**lau2019-nuts2016.xlsx** seems to be bad encoded as at least 16,383 columns found

```
in2csv --write-sheets - lau2019-nuts2016.xlsx

/home/wilke/.anaconda3/lib/python3.9/site-packages/agate/utils.py:285: UnnamedColumnWarning: Column 16383 has no name. Using "ddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd".

```

**Comment by adibaba** [comment]
**lau2020-nuts2016.xlsx**: at least 16,383 columns found

```
in2csv --write-sheets - lau2020-nuts2016.xlsx

/home/wilke/.anaconda3/lib/python3.9/site-packages/agate/utils.py:285: UnnamedColumnWarning: Column 16383 has no name. Using "ddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd".

```

**Comment by adibaba** [comment]
Tried to open xlsx in **Google Drive**: File to large.

**Comment by adibaba** [comment]
```
sudo apt-get install gnumeric

ssconvert -S lau2020-nuts2016.xlsx lau2020-nuts2016.csv

ls -1
lau2020-nuts2016.csv.0
lau2020-nuts2016.csv.1
lau2020-nuts2016.csv.10
lau2020-nuts2016.csv.11
lau2020-nuts2016.csv.12
lau2020-nuts2016.csv.13
lau2020-nuts2016.csv.14
lau2020-nuts2016.csv.15
lau2020-nuts2016.csv.16
lau2020-nuts2016.csv.17
lau2020-nuts2016.csv.18
lau2020-nuts2016.csv.19
lau2020-nuts2016.csv.2
lau2020-nuts2016.csv.20
lau2020-nuts2016.csv.21
lau2020-nuts2016.csv.22
lau2020-nuts2016.csv.23
lau2020-nuts2016.csv.24
lau2020-nuts2016.csv.25
lau2020-nuts2016.csv.26
lau2020-nuts2016.csv.27
lau2020-nuts2016.csv.28
lau2020-nuts2016.csv.29
lau2020-nuts2016.csv.3
lau2020-nuts2016.csv.30
lau2020-nuts2016.csv.31
lau2020-nuts2016.csv.32
lau2020-nuts2016.csv.33
lau2020-nuts2016.csv.34
lau2020-nuts2016.csv.35
lau2020-nuts2016.csv.36
lau2020-nuts2016.csv.37
lau2020-nuts2016.csv.38
lau2020-nuts2016.csv.39
lau2020-nuts2016.csv.4
lau2020-nuts2016.csv.40
lau2020-nuts2016.csv.41
lau2020-nuts2016.csv.42
lau2020-nuts2016.csv.43
lau2020-nuts2016.csv.44
lau2020-nuts2016.csv.45
lau2020-nuts2016.csv.46
lau2020-nuts2016.csv.47
lau2020-nuts2016.csv.5
lau2020-nuts2016.csv.6
lau2020-nuts2016.csv.7
lau2020-nuts2016.csv.8
lau2020-nuts2016.csv.9
lau2020-nuts2016.xlsx

```

**Comment by adibaba** [comment]
Sheet names:
```
in2csv --write-sheets - -n lau2020-nuts2016.xlsx 

Change log
Overview status
Overview POP_2020
BE NUTS 2021 and NUTS 2016
BG
CZ
DK
DE
EE NUTS 2021 and NUTS 2016
IE
EL
ES
FR
HR NUTS 2021 and NUTS 2016
IT NUTS 2021
IT NUTS 2016
CY
LV
LT
LU
HU
MT
NL
AT
PL
PT
RO
SI
SK
FI
SE
UK NUTS 2021 and NUTS 2016
IS
LI
NO
CH
ME
MK
AL
RS
TR
BA
XK
COMMENTS
Combined
Cities
Greater cities
FUAs
```

**Comment by adibaba** [comment]
solved and integrated in 4597492f4718a2f6bcfc47af8db33dc133754516
